storage/storage.go

```diff
@@ -23,7 +23,7 @@ func GetStorageProvider(providerType string, clientId string, clientSecret strin
 	case "AWS S3":
 		return NewAwsS3StorageProvider(clientId, clientSecret, region, bucket, endpoint), nil
 	case "MinIO":
-		return NewMinIOS3StorageProvider(clientId, clientSecret, "_", bucket, endpoint), nil
```

Comment:
The default value "_" should not be changed. You changed the default behavior

Reply:
Sorry, I still don't know how to fix this. Let me close this issue for now.

Reply:
@parasiteoflife yes, I like the latter one. So simple and keep compatibility